### DISCUSSION TITLE
`bilibili` video support

### DISCUSSION THREAD

**Discussion by imagicw** [nunocoracao/blowfish#1602] [Ideas]
Hi

Bilibili is the most fasmous video platform in China like Youtube, there are a lot videos in the site. So could you pls add a support for `bilibili` videos?

Bilibili Website: https://www.bilibili.com

Thank you!

**Comment by GrassBlock1** [comment]
maybe simply copy the embed code of the video (which is present in share memu of the video on desktop website) will work.
